charts/mlp/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
 - email: caraml-dev@caraml.dev
   name: caraml-dev
 name: mlp
-version: 0.4.20
+version: 0.4.21
```

Comment:
This can be a minor version upgrade. Good amount of configs moved and when merged the caraml chart will also need to update the default values and the deployments using these charts need to update their values files.

Reply:
Ah, makes sense. Updated.